app/helpers/action_links_helper.rb

```diff
@@ -50,6 +50,12 @@ def display_refund_link_for?(resource)
     can?(:refund, resource)
   end
 
+  def display_revert_link?
+    return false unless current_user
+
+    %w[agency_with_refund agency_super finance_admin finance finance_super].include?(current_user.role)
```

Comment:
The ability is bounded with the type of payment and we don't have it when we deal with this check. Open to suggestions :)

Reply:
I'd be up for making an additional generic `:view_revert_page` ability or something like that. Kind of like how we deal with the list of back office users. We have two types of users who can view that page, but they each have different abilities to actually manage the users.

Reply:
I was thinking about that, but then we will have an ability defined for the sake of a view, and it feels a bit wrong to me. Anyway, we have the same problem for payments now (we show the link to anyone as we can't check one ability) so I was thinking let's discuss this case on slack and fix them both in another PR

Reply:
If we have an ability for it then we can also check the authorization on the `index` view, so it would serve multiple uses. Happy to discuss more on Slack though.